### PR TITLE
Added swift-tools-support-core to update-checkout

### PR DESCRIPTION
**Pull request by hartbit**
Added `swift-tools-support-core` as the first step towards integrating it into CI.

### CODE REVIEW

**Comment by hartbit** [comment]
@swift-ci please smoke test

File: utils/update_checkout/update-checkout-config.json [hunk 1]
```diff
@@ -4,10 +4,12 @@
     "repos" : {
        "swift": {
             "remote": { "id": "apple/swift" } },
-       "cmark": {
+        "cmark": {
```
Comment (compnerd) [inline utils/update_checkout/update-checkout-config.json]: why the whitespace change?

**Review by compnerd — APPROVED** [review]

File: utils/update_checkout/update-checkout-config.json (hunk 1 above)
Comment (hartbit) [reply utils/update_checkout/update-checkout-config.json]: It was unintended. Fixed :)

**Review by aciidgh — APPROVED** [review]

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Pull request merged**